### PR TITLE
Address further security vulnerabilities

### PR DESCRIPTION
**Pull request by hepsimo**
Resolves:
BI-13409

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -25,11 +25,12 @@
         <api-helper-java.version>1.1.0-rc1</api-helper-java.version>
         <rest-service-common-library.version>0.0.3</rest-service-common-library.version>
         <environment-reader-library.version>1.3.2</environment-reader-library.version>
+        <jackson.version>2.15.3</jackson.version>
```
Comment (PhilipGlover) [inline pom.xml:28]: in the dependency report I still see `jackson-databind-2.13.15` as a Medium vulnerability
Screenshot 2023-11-06 at 10 56 08 am
it's coming from `aws-java-sdk-core:jar:1.12.343`

File: pom.xml [hunk 2]
```diff
@@ -25,11 +25,12 @@
...
         <!-- Logging -->
         <log4j-bom.version>2.17.0</log4j-bom.version>
         <structured-logging.version>1.9.12</structured-logging.version>
-        <avro.version>1.8.1</avro.version>
-        <ch-kafka.version>1.4.8</ch-kafka.version>
+        <avro.version>1.11.3</avro.version>
+        <ch-kafka.version>1.4.4</ch-kafka.version>
```
Comment (PhilipGlover) [inline pom.xml:33]: does ch-kafka need to be dropped back from 1.4.8 to 1.4.4 ?  I thought this was okay?

File: suppress.xml [hunk 3]
```diff
@@ -9,4 +9,54 @@
...
+        <packageUrl regex="true">^pkg:maven/org\.json/json@.*$</packageUrl>
+        <cve>CVE-2022-45688</cve>
+    </suppress>
+    <suppress until="2024-04-30Z">
+        <notes><![CDATA[
+   file name: json-20231013.jar
+   ]]></notes>
+        <packageUrl regex="true">^pkg:maven/org\.json/json@.*$</packageUrl>
+        <cve>CVE-2023-5072</cve>
+    </suppress>
```
Comment (PhilipGlover) [inline suppress.xml:53]: perhaps I'm doing something wrong, but I don't see `json-20231013.jar` listed as vulnerability.  I threw some `xxxx`'s in the package regex and it didn't show when I ran the check.

File: pom.xml (hunk 2 above)
Comment (hepsimo) [reply pom.xml:33]: Actually we needed newer versions because of HIGH vuln. in avro 1.8.1

File: suppress.xml (hunk 3 above)
Comment (hepsimo) [reply suppress.xml:53]: You're right, these json ones were added a bit earlier, but aren't required any more. I'll remove them.

**Review by PhilipGlover — APPROVED** [review]
- mvn verify green
- just structured logging left as expected
Screenshot 2023-11-06 at 3 16 17 pm

**Pull request merged**